roles/node_exporter/README.md

```diff
@@ -0,0 +1,28 @@
+Role Name
+=========
+
+Deploys a "node\_exporter" under the prometheus user
+
+Role Variables
+--------------
+
+None at the moment
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: username.rolename, x: 42 }
```

Comment:
This seems to be the standard placeholder. I think we should either fill that in properly or remove it.